thirdparty/gettext/CMakeLists.txt

```diff
@@ -13,6 +13,9 @@ ep_get_source_dir(SOURCE_DIR)
 ep_get_binary_dir(BINARY_DIR)
 
 set(CFG_ENV_VAR "CC=\"${CC}\"")
+if(${IS_ANDROID})
+    set(CFG_ENV_VAR "${CFG_ENV_VAR} && LDFLAGS=-lc")
```

Comment:
This breaks Android build as now on Android the `CC` variable won't be `${CC}` passed into CMake but raw default `gcc` instead.

```
set(CFG_ENV_VAR "${CFG_ENV_VAR} LDFLAGS=-lc")
```
should be OK.